include/ametsuchi/currency.h

```diff
@@ -18,34 +18,36 @@
 #ifndef AMETSUCHI_CURRENCY_H
 #define AMETSUCHI_CURRENCY_H
 
+#include <bits/stdc++.h>
 #include <stdint.h>
 #include <string>
 
 namespace ametsuchi {
 
 class Currency {
  public:
-  explicit Currency(uint64_t amount, uint8_t precision = 2);
+  explicit Currency(__int128_t amount, uint8_t precision = 2);
```

Comment:
Currency has __int128_t instead of uint64, because there is case using signed int.
and flatbuffer's scheme manage amount(string)